pages/[lang]/cart.js

```diff
@@ -11,23 +11,29 @@ import withCart from "containers/cart/withCart";
 import CartItems from "components/CartItems";
 import CheckoutButtons from "components/CheckoutButtons";
 import Link from "components/Link";
-import { Router } from "routes";
+import Layout from "components/Layout";
+import Router from "translations/i18nRouter";
 import PageLoading from "components/PageLoading";
-import track from "lib/tracking/track";
+// import track from "lib/tracking/track";
 import variantById from "lib/utils/variantById";
-import trackCartItems from "lib/tracking/trackCartItems";
-import TRACKING from "lib/tracking/constants";
+// import trackCartItems from "lib/tracking/trackCartItems";
+// import TRACKING from "lib/tracking/constants";
```

Comment:
Remove commented-out code if it’s not going to be used in the future.

Reply:
https://github.com/reactioncommerce/example-storefront/pull/667/commits/f99f9adf797a20148632641ef9413b915e711c56